.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Feature request
+about: Suggest an idea/feature for this project
+---
+<!-- **Are you in the right place?**
+1. For issues or feature requests, please create an issue in this repository.
+2. For general technical and non-technical questions, we are happy to help you on our [Rook.io Slack](https://Rook-io.slack.com).
+3. Did you already search the existing open issues for anything similar? -->
+
+Is this a bug report or feature request?
+* Feature Request
+
+**Feature Request**
+
+Are there any similar features already existing:
+
+What should the feature do:
```

Comment:
You can probably collapse this and the next one into "Describe the feature and the problem it will solve." I see no need for two separate fields.

Reply:
Depends on how you see it. I see those fields as a separate:
1. `What should the feature do`: What the feature really does, e.g. run Ceph as a storage backend.
2. `What would be solved through this feature`: What the feature "actually" solves, e.g. world storage needs are solved by adding Ceph as storage backend.

Reply:
So you mean for (2) to be a "Motivation" or "Use case" field. It should be renamed as such.